darwin/dataset/remote_dataset_v1.py

```diff
@@ -346,9 +345,10 @@ def complete(self, items: Iterator[DatasetItem]) -> None:
         items : Iterator[DatasetItem]
             The ``DatasetItem``\\s to be completed.
         """
-        wf_template_id_mapper = lambda item: item.current_workflow[
-            "workflow_template_id"
-        ]
+
+        def wf_template_id_mapper(item):
```

Comment:
interesting here that ruff hates lambdas. I don't disagree, the syntax sucks in python